test/test_browser.py

```diff
@@ -2968,6 +2968,10 @@ def test_glfw_events(self):
     self.btest('test_glfw_events.c', args=['-sUSE_GLFW=2', "-DUSE_GLFW=2", '-lglfw', '-lGL'], expected='1')
     self.btest('test_glfw_events.c', args=['-sUSE_GLFW=3', "-DUSE_GLFW=3", '-lglfw', '-lGL'], expected='1')
 
+  @requires_graphics_hardware
+  def test_glfw3_hi_dpi_aware(self):
+    self.btest('test_glfw3_hi_dpi_aware.c', args=['-sUSE_GLFW=3', '-lglfw', '-lGL'], expected='1')
```

Comment:
Can you use btest_exit here .. then you don't need any use of `REPORT_RESULT` in your code, you just return 0 from main.

Reply:
I just copy/pasted other glfw tests but easy to change. Done.